src/Model/Photo.php

```diff
@@ -476,6 +476,8 @@ public static function store(Image $image, int $uid, int $cid, string $rid, stri
 			'backend-ref' => $backend_ref
 		];
 
+		$fields = DI::dbaDefinition()->truncateFieldsForTable('photo', $fields);
```

Comment:
Shouldn't this be automatically called in `Database->update` and `Database->insert`?

Reply:
Since this will have a (small) impact on the performance, I would like to do this only at some dedicated places.